packages/accordions/src/utils/useStepContext.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { createContext, useContext } from 'react';
+
+export interface IStepContext {
+  currentStepIndex: number;
+}
+
+export const StepContext = createContext<IStepContext | undefined>(undefined);
+
+export const useStepContext = () => {
+  const context = useContext(StepContext);
+
+  if (context === undefined) {
+    throw new Error('useStepContext must be used within a StepContext.Provider');
```

Comment:
This error is meant to help consumers understand required nesting, but it's using wording/components that only affect our implementation.

Another wording may be "This component must be rendered within a `Stepper.Step` component"